apps/legacy-api/testing/LegacyApiTesting.ts

```diff
@@ -1,57 +1,20 @@
 import { LegacyStubServer, RegisteredRoute } from './LegacyStubServer'
-import { NestFastifyApplication } from '@nestjs/platform-fastify'
-import { InjectOptions, Response as LightMyRequestResponse } from 'light-my-request'
+import { ApiTesting } from '../../libs/rootserver/testing/ApiTesting'
 
 /**
  * LegacyApi Testing framework.
  */
-export class LegacyApiTesting {
+export class LegacyApiTesting extends ApiTesting {
   constructor (
-    private readonly stubServer: LegacyStubServer = new LegacyStubServer()
```

Comment:
The previous code has a `private` access modifier and this means the existing programmer is very specific with the accessibility of the variable.  After you refactor the code, you should try to use the most restrictive access modifier as much as you can.

In this context, it is impossible to use private but you still can use protected instead of default access modifier (Means you don't put anything) which is the public `access` modifier